modules/ROOT/partials/misc/ui-languages.adoc

```diff
@@ -1,45 +1,52 @@
 [[supported-languages]]
 == Supported Languages
 
+[IMPORTANT]
+====
+From {productname} 8.0, language codes use RFC5646 format with hyphens (-) instead of underscores (_). For backward compatibility, both formats are supported in {productname} 8, but underscore format will show console warning.
```

Comment:
```suggestion
From {productname} {productmajorversion}, language codes use RFC5646 format with hyphens (-) instead of underscores (_). For backward compatibility, both formats are supported in {productname} {productmajorversion}, but underscore format will show console warning.
```